projects/module-2/whats-cookin-part-one.md

```diff
@@ -0,0 +1,277 @@
+* [InteractJS](https://interactjs.io/){:target='blank'}
+
+</section>
+
+---
+
+### Iteration 1 - Get Familiar with the Data and Recipes
+
+Checkout the `data` directory and explore the data. Get a sense of what each property is and what the data nesting is like.  
+
+#### Recipe functionality
+
+You should have functions that:
+  * Return a filtered list of recipes based on a tag. (Extension option: filtering by multiple tags)
+  * Return a filtered list of recipes based on a recipe name. (Extension option: filtering by name or ingredients)
+  * Determine the names of ingredients needed for a given recipe.
+  * Calculate the cost of a given recipe's ingredients
+  * Return the directions / instructions for a given recipe
+
+<section class="note">
+
+### Calculating the cost of a Recipe
+
+To calculate the cost of a recipe, you will need to factor in the amount listed in the recipe. The price (`estimatedCostInCents`) of an ingredient is per `unit` - regardless of if that unit is a cup, teaspoon, etc.
+
+For example, if flour is marked as 100 cents, and the recipe calls for 1.5 cups, then it would cost 150 cents. Similarly, if a tomato is $2, and the recipe requires 2 tomatoes, then the cost would be $4.  
+
+Please note that the prices for the recipes might be higher than is realistic. That's okay - the mock data has high prices. You can always double check your math by manually calculating the price of a recipe - and of course, by writing thorough tests.
+</section>
+
+<section class="note">
+### Hint
+
+A common theme when dealing with data is not to manipulate the original data, but to create copies to work with.  
+
+You as the developer will need to make decisions about the shape of data you'll want back from various functions.  Consider how you will be using this data to update the DOM. 
+</section>
+
+#### User Stories (Dashboard)
+
+Use the `domUpdates.js` file to add information to the DOM. This JS file should call functions from `index.js`. There should not be any DOM manipulation within your `index.js` file.
+
+- As a user, I should be able to view all recipes.
+- As a user, I should be able to click on a recipe to view more information including directions, ingredients needed, and total cost.
+- As a user, I should be able to filter recipes by a tag. (Extension option: by multiple tags)
+- As a user, I should be able to search recipes by their name. (Extension option: by name or ingredients)
+
+---
+
+### Iteration 2 - Users
+
+#### User Data
+
+You should have functions that:
+- Allow a user to add/remove a recipe to/from their recipesToCook list (add to my `recipesToCook`)
+- Filter my `recipesToCook` by a tag. (Extension option: filter by multiple tags)
+- Filter my `recipesToCook` by its name. (Extension option: filter by name or ingredients)
+
+<!-- The last two functions above should be the same as 2 they already built, they shouldn't build a duplicate -->
```

Comment:
I think you can add an explicit note that this can be accomplished using (possibly refactored) functions from Iteration 1. It isn't SUPER clear, as is, that the functions from Iteration 1 apply here. I can see lots of students forgetting what they did in Iteration 1 and just writing new functions by the time they get to Iteration 2.

Reply:
Thank you - added a note